### PR TITLE
Controller logging

### PR DESCRIPTION
**Pull request by cgwalters**
A few patches for the node controller to help me debug https://github.com/openshift/machine-config-operator/pull/385

### CODE REVIEW

File: pkg/controller/node/node_controller.go [hunk 1]
```diff
@@ -376,7 +376,13 @@ func (ctrl *Controller) syncMachineConfigPool(key string) error {
 	}
 
 	if machineconfigpool.Status.Configuration.Name == "" {
-		return fmt.Errorf("Empty Current MachineConfig")
+		delay := 5 * time.Second
+		// Previously we spammed the logs about empty pools.
+		// Let's just pause for a bit here to let the renderer
+		// initialize them.
+		glog.Infof("Pool %s is unconfigured, pausing %v for renderer to initialize", delay)
+		time.Sleep(delay)
```
Comment (runcom) [inline pkg/controller/node/node_controller.go]:
```
# github.com/openshift/machine-config-operator/pkg/controller/node
pkg/controller/node/node_controller.go:383: Infof format %v reads arg #2, but call has 1 arg
pkg/controller/node/status.go:32: Infof format %s has arg newPool.Status of wrong type github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1.MachineConfigPoolStatus
```

Comment (cgwalters) [reply pkg/controller/node/node_controller.go]: OK I dropped the status patch, did that at the last second and hadn't tested it.  Doing that "right" is probably changing the operator to do the logging, as it already has the code to render things to a string.

**Comment by ashcrow** [comment]
```
'namespaces "ci-op-cilrrn4d" not found' (will not retry!)
```

/test rhel-images

**Comment by cgwalters** [comment]
OK this is passing tests and looks a bit nicer:

```
I0213 14:39:10.877232       1 start.go:46] Version: 3.11.0-619-g7ced8fb4-dirty
I0213 14:39:11.077210       1 leaderelection.go:187] attempting to acquire leader lease  openshift-machine-config-operator/machine-config-controller...
E0213 14:39:11.577610       1 event.go:259] Could not construct reference to: '&v1.ConfigMap{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"machine-config-controller", GenerateName:"", Namespace:"openshift-machine-config-operator", SelfLink:"/api/v1/namespaces/openshift-machine-config-operator/configmaps/machine-config-controller", UID:"209b86ab-2f9d-11e9-befd-0ed171f9d180", ResourceVersion:"5454", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:63685665551, loc:(*time.Location)(0x1dbcac0)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string{"control-plane.alpha.kubernetes.io/leader":"{\"holderIdentity\":\"machine-config-controller-7bc86b94d-4sqnt_203d8ff8-2f9d-11e9-891d-0a580a81000c\",\"leaseDurationSeconds\":90,\"acquireTime\":\"2019-02-13T14:39:11Z\",\"renewTime\":\"2019-02-13T14:39:11Z\",\"leaderTransitions\":0}"}, OwnerReferences:[]v1.OwnerReference(nil), Initializers:(*v1.Initializers)(nil), Finalizers:[]string(nil), ClusterName:""}, Data:map[string]string(nil), BinaryData:map[string][]uint8(nil)}' due to: 'no kind is registered for the type v1.ConfigMap in scheme "github.com/openshift/machine-config-operator/cmd/common/helpers.go:39"'. Will not report event: 'Normal' 'LeaderElection' 'machine-config-controller-7bc86b94d-4sqnt_203d8ff8-2f9d-11e9-891d-0a580a81000c became leader'
I0213 14:39:11.577713       1 leaderelection.go:196] successfully acquired lease openshift-machine-config-operator/machine-config-controller
I0213 14:39:11.877474       1 template_controller.go:111] Starting MachineConfigController-TemplateController
I0213 14:39:12.077382       1 kubelet_config_controller.go:138] Starting MachineConfigController-KubeletConfigController
I0213 14:39:12.078366       1 render_controller.go:112] Starting MachineConfigController-RenderController
I0213 14:39:12.477855       1 node_controller.go:121] Starting MachineConfigController-NodeController
I0213 14:39:17.877755       1 node_controller.go:383] Pool worker is unconfigured, pausing 5s for renderer to initialize
I0213 14:39:17.977485       1 node_controller.go:383] Pool master is unconfigured, pausing 5s for renderer to initialize
I0213 14:39:18.477993       1 render_controller.go:456] Generated machineconfig worker-acfb16efd24e16e8e7cf41818e246bed from 3 configs: [{MachineConfig  00-worker  machineconfiguration.openshift.io/v1  } {MachineConfig  00-worker-ssh  machineconfiguration.openshift.io/v1  } {MachineConfig  01-worker-kubelet  machineconfiguration.openshift.io/v1  }]
I0213 14:39:19.078891       1 render_controller.go:456] Generated machineconfig master-2f8dafc81a861ef2cc42845dee9d8b4b from 3 configs: [{MachineConfig  00-master  machineconfiguration.openshift.io/v1  } {MachineConfig  00-master-ssh  machineconfiguration.openshift.io/v1  } {MachineConfig  01-master-kubelet  machineconfiguration.openshift.io/v1  }]
I0213 14:39:24.378373       1 node_controller.go:432] Setting node ip-10-0-40-39.ec2.internal to desired config master-2f8dafc81a861ef2cc42845dee9d8b4b
I0213 14:40:56.541745       1 node_controller.go:432] Setting node ip-10-0-168-88.ec2.internal to desired config worker-acfb16efd24e16e8e7cf41818e246bed
I0213 14:41:01.681050       1 node_controller.go:432] Setting node ip-10-0-168-88.ec2.internal to desired config worker-acfb16efd24e16e8e7cf41818e246bed
I0213 14:41:06.777993       1 node_controller.go:432] Setting node ip-10-0-159-39.ec2.internal to desired config worker-acfb16efd24e16e8e7cf41818e246bed
I0213 14:41:11.800841       1 node_controller.go:432] Setting node ip-10-0-168-88.ec2.internal to desired config worker-acfb16efd24e16e8e7cf41818e246bed
I0213 14:42:21.209958       1 node_controller.go:432] Setting node ip-10-0-132-101.ec2.internal to desired config worker-acfb16efd24e16e8e7cf41818e246bed
```

**Comment by ashcrow** [comment]
Router related

/test e2e-aws

**Pull request merged**